src/app/downloads/downloads-table.component.css

```diff
@@ -23,17 +24,21 @@ td {
   white-space: nowrap;
 }
 table.sticky th, table.sticky td {
-  min-width: 150px;
-  max-width: 150px;
+  min-width: 200px;
+  max-width: 200px;
   border-right: none;
   text-overflow: ellipsis;
   overflow: hidden;
   text-align: left;
 }
 .scroll-x-wrapper {
-  max-width: calc(100% - 150px);
+  flex-grow: 1;
+  max-width: calc(100% - 200px);
   overflow-x: auto;
 }
+.scroll-x-wrapper > table {
```

Comment:
With this 100% wide, I dunno where we're gonna put the expandable report when this turns into the summary table with Gigi's ticket #119, but that's a design problem I'm just not getting very far with in discussion. I feel lucky I was able to convince them we shouldn't remove the report, shrug.

Reply:
That's to the left of both the chart and table, right?  Or maybe that's changed, per your comments.

Yeah, I think all of this table CSS will have to be readdressed in that ticket.  I thought about rewriting this whole table in flexbox ... but the possibility of failure was too much for me!

Reply:
Verbally, we discussed that the expandable report would expand to the right of the summary table. The summary table will take the place of this report and the report is the expandable bit. I think we are gonna want to redo the table in flexbox to make adaptive layout easier to deal with, but no need to do that today.